WcaOnRails/app/helpers/results_helper.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module ResultsHelper
+  def solve_tds_for_result(result)
+    trimmed_indices = result.trimmed_indices
+    best_index = result.best_index
+    worst_index = result.worst_index
```

Comment:
Sure thing! I just moved the code out of the results table (probably written by Tim) without in-depth analysis.